### PR TITLE
Upgrade to govuk frontend 5.1

### PR DESCRIPTION
**Pull request by MuriloDalRi**
https://trello.com/c/qmPB0DEN/3581-upgrade-our-apps-to-v51-of-the-design-system-3

### CODE REVIEW

File: Gemfile.lock
Comment (deborahchua) [inline Gemfile.lock:1]: Should the changes in this file be in a separate commit, so that this commit only contains the move to es6-components? E.g. like [this commit](https://github.com/alphagov/collections/pull/3648/commits/64e2be112c8d06cdd9209b1f47f0b5ba7526f44e)

Comment (MuriloDalRi) [reply Gemfile.lock:1]: Are you referring to the first or second commit?

Comment (deborahchua) [reply Gemfile.lock:1]: Oh sorry I meant the second commit - I pressed the wrong review button!

Comment (MuriloDalRi) [reply Gemfile.lock:1]: I guess that's up to personal preference, I just didn't see the need for that. Can do that if you think it's needed though?

**Review by deborahchua — APPROVED** [review]
